pkg/api/componentreadiness/component_report.go

```diff
@@ -929,6 +483,22 @@ func (c *componentReportGenerator) getTestStatusFromBigQuery(ctx context.Context
 		}
 
 	}()
+
+	// TODO: temporary hardcode these overrides, move to a new config file later.
+	/*
+		junitTableOverrides := []crtype.JunitTableOverride{
+			{
+				VariantKey:          "JobTier",
+				VariantValue:        "rare",
+				TableName:           "junit_rarely_run",
+				RelativeStartOffset: "90d",
+			},
+		}
+
+	*/
+	// TODO: Perform additional parallel queries for any table overrides based on the included variants,
+	// and merge the results.
```

Comment:
This is the portion I started on for rarely run jobs, I will resume here after merging if we can live with this disabled for the time being.